documentation/autoinstall-reference.md

```diff
@@ -157,24 +157,28 @@ Identifier of the source to install (e.g., `"ubuntu-server-minimized"`).
 **default:** DHCP on interfaces named eth\* or en\*
 **can be interactive:** yes
 
-[netplan](https://netplan.io/reference) formatted network configuration. This will be applied during installation as well as in the installed system. The default is to interpret the config for the install media, which runs DHCPv4 on any interface with a name matching "eth\*" or "en\*" but then disables any interface that does not receive an address.
+[Netplan-formatted](https://netplan.io/reference) network configuration. This will be applied during installation as well as in the installed system. The default is to interpret the config for the install media, which runs DHCPv4 on any interface with a name matching "eth\*" or "en\*" but then disables any interface that does not receive an address.
 
-For example, to run dhcp6 on a particular NIC:
+For example, to run DHCPv6 on a particular NIC:
 
-    network:
-      version: 2
-      ethernets:
-        enp0s31f6:
-          dhcp6: yes
+```yaml
+network:
+  version: 2
+  ethernets:
+    enp0s31f6:
+      dhcp6: yes
+```
 
-Note that thanks to a bug, the version of subiquity released with 20.04 GA forces you to write this with an extra "network:" key like so:
+Note that thanks to a bug, the version of Subiquity released with 20.04 GA forces you to write this with an extra `network:` key like so:
```

Comment:
Unrelated to the change so please feel free to ignore.
Doesn't "thanks to a bug" sound odd? How about "because of a bug"?